Documentation/ReleaseNotes.md

```diff
@@ -266,12 +266,6 @@ Enabling this feature on other versions of Unity may result in incorrect capture
 
 When loading an MRTK based project, the MRTK Configurator dialog may display multiple times. This is related to MRTK detecting multiple loads of the project. This issue will be investigated and addressed in a future version oF the MRTK.
 
-**The type or namespace name 'TrackedPoseDriver' could not be found**
```

Comment:
Should we leave this for reference, just in case somebody happens to get in this state?

Reply:
we can.

Reply:
added back